cicecore/cicedynB/dynamics/ice_transport_remap.F90

```diff
@@ -632,12 +679,21 @@ subroutine horizontal_remap (dt,                ntrace,     &
 
       endif  ! nghost
 
-      !--- tcraig, tcx, this omp loop leads to a seg fault in gnu
+#ifdef OMP_TIMERS
+      if (my_task == master_task) then
+         time1 = OMP_GET_WTIME()
+         write(nu_diag,*) trim(subname),OMP_GET_THREAD_NUM(),iblk,time1-time0,'t5'
+         time0 = time1
+         call flush_fileunit(nu_diag)
+      endif
+#endif
+
       !--- need to check private variables and debug further
-      !$TCXOMP PARALLEL DO PRIVATE(iblk,i,j,ilo,ihi,jlo,jhi,this_block,n,m, &
-      !$TCXOMP                     edgearea_e,edgearea_n,edge,iflux,jflux, &
-      !$TCXOMP                     xp,yp,indxing,indxjng,mflxe,mflxn, &
-      !$TCXOMP                     mtflxe,mtflxn,triarea,istop,jstop,l_stop)
+      !$OMP PARALLEL DO PRIVATE(iblk,i,j,ilo,ihi,jlo,jhi,this_block,n,  &
+      !$OMP                     edgearea_e,edgearea_n,edge,iflux,jflux, &
+      !$OMP                     xp,yp,indxing,indxjng,mflxe,mflxn, &
+      !$OMP                     mtflxe,mtflxn,triarea,istop,jstop,l_stop) &
+      !$OMP SCHEDULE(runtime)
```

Comment:
same here, if it works.

Reply:
Same here.  I check both the file diffs in the PR and the code in my sandbox.